### PR TITLE
[FIX] Network File: handle pajek bug & two small fixes

### PR DESCRIPTION
**Pull request by jerneju**
##### Issue
Reading `pajek` format may fail.

##### Description of changes
Do not crash a program due to that reading (parse) error.

##### Includes
- [X] Code changes
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by jerneju** [comment]
https://sentry.io/biolab/orange3-network/issues/437792064/

File: orangecontrib/network/readwrite.py [hunk 1]
```diff
@@ -295,7 +300,8 @@ def read_pajek(path, encoding='UTF-8', project=False, auto_table=False):
     for node in G.node:
         G.node[node]['label'] = node
     nx.relabel_nodes(G, remapping, copy=False)
-    assert not table or len(table) == G.number_of_nodes(), 'There was a bug in NetworkX. Please update to git if need be'
+    if not table or len(table) == G.number_of_nodes():
+        raise PajekBug("There was a bug in NetworkX. Please update to git if need be")
```
Comment (kernc) [inline orangecontrib/network/readwrite.py]: "There is a bug in your version of NetworkX reading Pajek files. Please update your NetworkX installation."

**Comment by thocevar** [comment]
@kernc, this merge breaks reading all our "documentation networks" such as airtrafic or lastfm. I'm using the latest 2.2rc1.dev version of networkx.

The conversion from "assert" to "if" in "read_pajek" should probably negate the condition.

**Comment by kernc** [comment]
Correct, fixed.

**Pull request merged**